File: video/src/main/java/com/zhuliyi/video/mvp/model/bean/VideoBaseResponse.java
package com.zhuliyi.video.mvp.model.bean;

import java.io.Serializable;

/**
 * Describe : baseResponse
 * Author : zhuly
 * Date : 2019-05-15
 */
public class VideoBaseResponse<T> implements Serializable {
    private String status;
    private T data;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public T getData() {
        return data;
    }

    public void setData(T data) {
        this.data = data;
    }

    public boolean isSuccess(){
        return "200".equals(status);
    }
}

File: video/src/main/java/com/leory/badminton/video/mvp/model/bean/VideoBaseResponse.kt
package com.leory.badminton.video.mvp.model.bean

import java.io.Serializable

/**
 * Describe : baseResponse
 * Author : zhuly
 * Date : 2019-05-15
 */
data class VideoBaseResponse<T>(var status: String? = null,
                                var data: T? = null) : Serializable {


    val isSuccess: Boolean
        get() = "200" == status
}
